src/main.js

```diff
@@ -37,6 +42,33 @@ async function run() {
         body += `\n:no_entry: ${errorMessage}`;
       }
 
+      const updateGitHubComment = commentId =>
+      octokit.issues.updateComment({
+        repo: github.context.repo.repo,
+        owner: github.context.repo.owner,
+        comment_id: commentId,
+        body,
+      })
```

Comment:
Move this within the if check in line 64? That way its not unnecessary executed when updateComment is `false`